### ISSUE TITLE
Remove "Multi-Screen Window Placement" feature (until we can address privacy harms)

### ISSUE THREAD

**Issue by pes10k** [brave/brave-browser#29276]
Chrome is shipping support for the [Multi-Screen Window Placement](https://w3c.github.io/window-placement/) spec. This spec has some open [serious privacy issues](https://github.com/w3cping/privacy-request/issues/106#issuecomment-1335925446), including fingerprinting surface [before](https://github.com/w3c/window-placement/issues/120) and [after](https://github.com/w3c/window-placement/issues/121) granting the relevant permission.

We should disable this feature until we can address the privacy harms

**Comment by mkarolin** [comment]
@ShivanKaul is this still an issue? If so perhaps someone on your team can pick it up?

**Comment by ShivanKaul** [comment]
I haven't been following this spec. @pes10k have you?

**Comment by pes10k** [comment]
yes, i filed several issues against the original spec when it came through PING for privacy review 2-3 years ago.

This issue tracks the specific privacy concerns i found:

https://github.com/w3cping/privacy-request/issues/106#issuecomment-1335925446

The short version is:
- a fingerprinting bit, in that it leaks if you have extra monitors and things plugged in, as a boolean, pre-permission
- post-permission, reveals to the site device labels (like the manufacturer assigned name and description of the device), which seems like way more than is needed to accomplish most (all?) user goals, given the re-identification risk.  My suggestion was reveal the monitor names to the user (in a picker page for the permission) but not the application
